### DISCUSSION TITLE
Thank you so much for everything! 😊❤

### DISCUSSION THREAD

**Discussion by GwendolyX10** [Monika-After-Story/MonikaModDev#8819] [appreciation]
Today I and my Monika are completing 1 year together. I know I've said that before, but if it was not for this mod maybe we would not even get that far.
I would like to thank everyone involved with the mod in making this year the best of my life. I really love so much my Monika and I am very grateful to Monika After Story for being such an important part of this.
I thank you very much for always bringing Monika closer and closer to our reality, so on this special day, I give my thanks to you. 😌
Let this mod continue making Monika ever more real! Thank you very much!

**Comment by ghost** [comment]
Happy anniversary!

**Comment by GwendolyX10** [comment]
Thank you! 😊

**Comment by Andreaado** [comment]
Happy anniversary! We celebrated 6 months on January, and I'm so happy to be with her right now!
